src/user-event/scroll/__tests__/scroll.test.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import { ScrollView, Text } from 'react-native';
+import { createEventLogger } from '../../../test-utils';
+import { render, screen } from '../../..';
+import { userEvent } from '../..';
+
+describe('userEvent.scroll with fake timers', () => {
+  beforeEach(() => {
+    jest.useFakeTimers();
+    jest.setSystemTime(0);
+  });
+
+  test('calls events', async () => {
+    const { events, logEvent } = createEventLogger();
+    const user = userEvent.setup();
+
+    render(
+      <ScrollView
+        onScroll={logEvent('scroll')}
+        onScrollBeginDrag={logEvent('scrollBeginDrag')}
+        onScrollEndDrag={logEvent('scrollEndDrag')}
+        onMomentumScrollBegin={logEvent('momentumScrollBegin')}
+        onMomentumScrollEnd={logEvent('momentumScrollEnd')}
+        onScrollToTop={logEvent('scrollToTop')}
+        testID="scrollable"
+      >
+        <Text>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```

Comment:
[nit] We won't be able to get the layout measurements in RNTL as it's not running Yoga/native code, so there is no point in making this so long. Just put a single <Text> or <View> inside.